### PR TITLE
Compatible with MW 1.35

### PR DESCRIPTION
**Pull request by RazeSoldier**

### CODE REVIEW

File: includes/UID.php [hunk 1]
```diff
@@ -35,7 +37,7 @@ public static function fromBin($value) {
 	}
 
 	public static function generate() {
-		$hex = \UIDGenerator::newTimestampedUID88(16);
+		$hex = MediaWikiServices::getInstance()->getGlobalIdGenerator()->newTimestampedUID128(16);
```
Comment (nbdd0121) [inline includes/UID.php:40]: This changes the number of bits from 88 to 128? Does the time extraction still work?

**Comment by nbdd0121** [comment]
Seems GlobalIdGenerator still has a newTimestampedUID88, should that be used instead?

**Comment by RazeSoldier** [comment]
You are right. That is my mistake. I will commit a new patch to fix it.

**Comment by nbdd0121** [comment]
That's a lot, as I prematurely merged this PR you probably need to open a new PR.

**Pull request merged**